src/components/ShotForm/ShotForm.js

```diff
@@ -8,54 +8,86 @@ import FormError from "../FormError";
 import http from "../../utils/http";
 import Input from "../Input";
 import Layout from "../Layout";
+import Loader from "../Loader";
+import Select from "../Select";
 
 const ShotForm = props => {
-  const { errors, handleChange, handleSubmit, values } = useFormik({
+  const [isLoading, setIsLoading] = useState(false);
+
+  const options = [
+    { value: "4", label: "4 Iron" },
+    { value: "5", label: "5 Iron" },
+    { value: "6", label: "6 Iron" },
+    { value: "7", label: "7 Iron" },
+    { value: "8", label: "8 Iron" },
+    { value: "9", label: "9 Iron" },
+    { value: "PW", label: "Pitching Wedge" }
+  ];
+
+  const {
+    errors,
+    handleChange,
+    handleSubmit,
+    touched,
+    setFieldValue,
+    values
+  } = useFormik({
     initialValues: {
-      club: "",
+      club: null,
       distance: ""
     },
     onSubmit: async values => {
       try {
-        await http.post("/shots", values);
+        await http.post("/shots", { ...values, club: values.club.value });
+        setIsLoading(true);
         props.history.push("/");
       } catch {}
```

Comment:
Probably should have a `finally` block here to set loading to false if there were any issues with the request so the user can continue on?